.github/workflows/agw-workflow.yml

```diff
@@ -213,6 +213,17 @@ jobs:
         run: |
           echo "Available storage:"
           df -h
+      - name: Notify failure to slack
+        if: failure() && github.event_name == 'push'
+        uses: rtCamp/action-slack-notify@v2.2.0
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_BAZEL_CI }}
+          SLACK_TITLE: "C/C++ unit tests with Bazel"
+          SLACK_USERNAME: "agw-workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+          SLACK_FOOTER: ' '
+          MSG_MINIMAL: actions url,commit
```

Comment:
Should be comma separated, I guess.

Reply:
`actions url` is one item. I tried to do it the way you commented, but ` url` is needed too. You can check the difference in #bazel-ci.

Reply:
Thanks for the clarification. I have misread the documentation and was confused by a line break there.